scripts/perftune.py

```diff
@@ -1335,32 +1335,36 @@ def dump_config(prog_args):
     sys.exit(0)
 
 try:
-    tuners = []
-
-    if TuneModes.disks.name in args.tune:
-        tuners.append(DiskPerfTuner(args))
-
-    if TuneModes.net.name in args.tune:
-        tuners.append(NetPerfTuner(args))
-
-    if TuneModes.system.name in args.tune:
-        tuners.append(SystemPerfTuner(args))
-
-    # Set the minimum mode among all tuners
-    if not args.irq_cpu_mask:
-        mode = PerfTunerBase.SupportedModes.combine([tuner.mode for tuner in tuners])
-        for tuner in tuners:
-            tuner.mode = mode
+    nics=args.nic
+    for nic in nics:
```

Comment:
I think this is just wrong. 
What you do here is you simulate running perftune.py for each of given nics (and all the rest of parameters) one after another. 
While what you should be doing is to do the same as if given NICs are a part of a bonding interface.
Namely you should have calculated the combined tuning mode for ALL nics and the rest of tunables, and then tune everything in a single take.